### ISSUE TITLE
Query String for cache-busting

### ISSUE THREAD

**Issue by eltimn** [Shadowfiend/sbt-resource-management#2]
First of all, thanks for publishing this. I've been working on something that would be similar to this (I was basing it on [Jammit](http://documentcloud.github.com/jammit/)), but haven't gotten very far into it and I think I'll just use this instead.

However, there is one small thing I'd like to address: some proxies don't cache items with a query string. Quoting [google](https://developers.google.com/speed/docs/best-practices/caching#LeverageProxyCaching) in the recommendations section:

> Don't include a query string in the URL for static resources.
> Most proxies, most notably Squid up through version 3.0, do not cache resources with a "?" in their URL even if a Cache-> control: public header is present in the response. To enable proxy caching for these resources, remove query strings from references to static resources, and instead encode the parameters into the file names themselves.

Basically, you need to rename the file instead of using a query string for it to be cached when behind one of those proxies. This probably represents a small number of users, but it may still be worth addressing.

What are your thoughts on this?
